core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -835,7 +831,7 @@ public void run() {
                     String contentTitle = info.playable.getFeedTitle();
                     Notification notification = null;
 
-                    NotificationCompat.Builder notificationBuilder = new android.support.v7.app.NotificationCompat.Builder(
+                    NotificationCompat.Builder notificationBuilder = new NotificationCompat.Builder(
```

Comment:
On which Android version did you test it? Didn't see any "upset" in 4.4
Also, not really sure I changed anything. As far as I can tell, there is only one NotificationCompat and the part I removed is the same we are importing anyway. Don't really see any difference